examples/python/helloworld/helloworld.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright 2015 gRPC authors.
```

Comment:
1. Should we copy the proto file, or reuse the existing one?
2. If we are copying an old file, what should the copy right header be?

Reply:
I *could* just reference the existing one, but that would require a less simple syntax in the import. Since this is the hello world, I wanted to go for the simpler variant. I considered doing a symlink, but I remembered that Piper would choke on that.

The precedent of books would suggest that you actually print both years -- the year of the original copyright as well as the year of the current production. However, I'm pretty sure our copyright check script would choke on that. I think keeping the original year is best.

Reply:
It looks good to me to keep another copy.